vispy/color/color_array.py

```diff
@@ -124,9 +128,24 @@ class ColorArray(object):
     Under the hood, this class stores data in RGBA format suitable for use
     on the GPU.
     """
-    def __init__(self, color='black', alpha=None, clip=False):
+    def __init__(self, color=None, color_space='rgb',
+                 alpha=None, clip=False):
```

Comment:
be careful about backward compat, the new arg should go at the end not be inserted earlier